src/JustSaying/Messaging/Channels/SubscriptionGroups/ISubscriptionGroupCollection.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.Messaging.Interrogation;
+
+namespace JustSaying.Messaging.Channels.SubscriptionGroups
+{
+    /// <summary>
+    /// Represents a collection of <see cref="ISubscriptionGroup"/> that can be started together.
+    /// </summary>
+    public interface ISubscriptionGroupCollection : IInterrogable
```

Comment:
That was my initial thought, but I think keeping them separate makes it easier in the future. Otherwise when the next person comes to change one they will need to split it up themselves, and have to work out where needs one/the other or it will grow into an interface that is both.
I would prefer separate interfaces for logically different constructs.

Reply:
I still don't think it makes sense. They will probably never need to diverge anyway. Think about `Action<T>`: when you combine many of them, by doing `var combined = action1 + action2;` the result is still of type `Action<T>` (not `CombinedAction<T>`). From the consumer's perspective it is important to hide the implementation detail from them.

Reply:
Happy to change it if @gkinsman agrees too?

Reply:
Have updated them both to use `ISubscriptionGroup`